spec/features/merchant_items/show_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe 'User story 34' do
```

Comment:
Great test - we might want to look into using a gem for data such as 'faker'. I ran into this when I was testing. Our application is supposed to be using large data sets and using faker would make it easy and quick to load data for testing and seeds. We dont have to - just a thought

Reply:
I agree with this, do we need to seed? I will look into faker. It will make it much easier as we move along